src/WarcraftLegacies.Source/Quests/Sentinels/QuestFeathermoon.cs

```diff
@@ -34,23 +29,20 @@ public QuestFeathermoon(Capital feathermoon) : base("Shores of Feathermoon",
 
     /// <inheritdoc />
     protected override string RewardDescription => 
-      $"Gain control of all units in Feathermoon Stronghold and learn to train Naisha from the {GetObjectName(UNIT_E00R_ALTAR_OF_WATCHERS_SENTINEL_ALTAR)}";
+      $"Gain the ability to train Naisha from the {GetObjectName(UNIT_E00R_ALTAR_OF_WATCHERS_SENTINEL_ALTAR)}";
```

Comment:
```suggestion
      $"Learn to train Naisha from the {GetObjectName(UNIT_E00R_ALTAR_OF_WATCHERS_SENTINEL_ALTAR)}";
```
Just how things have been standardized.